pkgs/by-name/pw/pwsafe-cli/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib,
+stdenv,
+fetchFromGitHub,
+testers,
+autoconf,
+automake,
+libX11,
+libXmu,
+withX11 ? true,
+openssl,
+}:
+
+let
+  baseVersion = "0.2.2beta";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pwsafe-cli";
+  version = baseVersion + "-unstable-2022-01-05";
+
+  src = fetchFromGitHub {
+    owner = "nsd20463";
+    repo = "pwsafe";
+    rev = "4bf61c5e7a5ce11cb30cd4cc4ce10291ee12ded9";
+    hash = "sha256-SXYN1aHuLwOvvqTSGZgFeVrAFBCRwYd3qpcnsc/VcXw=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
```

Comment:
Can we use `autoreconfHook`?
https://nixos.org/manual/nixpkgs/stable/#setup-hook-autoconf

Reply:
thx -- missed this.